clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -809,4 +809,7 @@ def warn_android_unversioned_fallback : Warning<
 
 def err_drv_triple_version_invalid : Error<
   "version '%0' in target triple '%1' is invalid">;
+
+def warn_missing_include_dirs : Warning <
```

Comment:
nit: no space between `Warning` and `<`, and two spaces for indentation on the next line

This makes the warning enabled by default. I'm not sure whether that's desirable. Maybe it should be under `-Wall` or `-Wextra` (what does gcc do?)

To achieve that, add the `DefaultIgnore` mark to this definition, and add `InGroup<All>` or `Extra` to the `DiagGroup` definition.

Reply:
Tested with GCC, its not enabled in `-Wall` or `-Wextra`, leaving as `DefaultIgnore` for now.